CHANGES.txt

```diff
@@ -35,6 +35,9 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       issue affected CommandGeneratorAction function actions specifically.
     - Added support for the PCH environment variable to support subst generators. 
 
+  From Daniel Moody:
```

Comment:
? shouldn't this be NINJA_FORCE_SCONS_BUILD

Reply:
fixed incorrect push mistake